pkg/apis/work/v1alpha2/binding_types.go

```diff
@@ -161,6 +218,11 @@ type BindingSnapshot struct {
 
 // ResourceBindingStatus represents the overall status of the strategy as well as the referenced resources.
 type ResourceBindingStatus struct {
+	// SchedulerObservedGeneration is the generation(.metadata.generation) observed by the scheduler.
+	// If SchedulerObservedGeneration is less than the generation in metadata means the scheduler hasn't confirmed
+	// the scheduling result or hasn't done the schedule yet.
+	// +optional
+	SchedulerObservedGeneration int64 `json:"schedulerObservedGeneration,omitempty"`
```

Comment:
So what if the evicition task has not been done yet but there comes a scaling-up event, how to deal with this situation by using `SchedulerObservedGeneration`?

Reply:
I'm not sure if I understand you correctly. 

A general idea in my mind is that `scheduler` shouldn't schedule the resource(in binding) to a cluster which already in one of the `graceful eviction tasks`.